### DISCUSSION TITLE
欢迎反馈 错误的配置信息 或 错误的独立DLC信息

### DISCUSSION THREAD

**Discussion by pjy612** [pjy612/SteamManifestCache#4] [Announcement]
如果发现在入库 app 时，出现错误的捆绑包信息，欢迎反馈，这边会安排修复配置信息

**Comment by alvisfei86** [comment]
好的~

**Comment by heyong5454** [comment]
为啥说我删库跑路了呢？跑路是没有，只是删库了而已。。

**Reply by pjy612** [reply]
？？？。。。没 只是当时你那个库 造福了挺多

**Reply by heyong5454** [reply]
没办法，我也不想关的，太多因素了。
最主要就是非法使用，太多了。。

**Reply by pjy612** [reply]
是的 当时 在 wxy 消失后，你可谓是 一库拖全部 2333。当时很多入库工具 全在白嫖你的库 23333

**Reply by ZEERDEER** [reply]
勇哥

**Comment by Avant-Garde0** [comment]
想问一下我使用了七次，昨天申请了一个环世界的dlc为什么今天被拒绝访问了,环世界的算错误的独立DLC是吗
